src/main/java/com/keniu/dto/UserLoginDto.java

```diff
@@ -0,0 +1,16 @@
+package com.keniu.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * Data Transfer Object (DTO) for
+ * representing the login response for a user in the books shop application.
+ * This class is used to transfer the authentication token between the server and the client
+ * after a successful login.
+ */
+@AllArgsConstructor
+@Data
+public class UserLoginDto {
+    private String token;
+}
```

Comment:
You can make it `record`
```suggestion
public record UserLoginResponseDto(String token) {
}
```

Reply:
Ok, done